docs/developer/http-api.md

```diff
@@ -4,6 +4,8 @@ Drand provides a JSON HTTP interface that clients can use to fetch randomness fr
 
 All that's required is the address of the HTTP interface and way to fetch from HTTP, e.g. `curl`.
 
+## Public endpoints
+
 The public [League of Entropy](https://blog.cloudflare.com/league-of-entropy/) HTTP APIs are available at:
```

Comment:
```suggestion
The public [League of Entropy](https://leagueofentropy.com/) HTTP APIs are available at:
```

Reply:
@jsoares I these around because I think we will need to change all these links in bulk (and it's not the scope of this PR). We will have to discuss what to link, if anything.

Reply:
"Revise the /developer section" is a pretty broad scope but sure, we can have a separate PR.